api/erc20.go

```diff
@@ -0,0 +1,108 @@
+package api
+
+import (
+	"fmt"
+	"net/http"
+
+	"github.com/julienschmidt/httprouter"
+
+	erc20ext "github.com/threefoldtech/rivine-extension-erc20"
+	"github.com/threefoldtech/rivine/pkg/api"
+	types "github.com/threefoldtech/rivine/types"
+)
+
+type (
+	// PluginGetERC20RelatedAddress contains the requested ERC20-related addresses.
+	PluginGetERC20RelatedAddress struct {
+		TFTAddress   types.UnlockHash   `json:"tftaddress"`
+		ERC20Address types.ERC20Address `json:"erc20address"`
```

Comment:
Wrong import type, you should use the type provided under `github.com/threefoldtech/rivine-extension-erc20/types` instead. If you didn't move it to there yet, you should. It doesn't make sense to have the ERC20 Address under `types` in rivine.

Reply:
This comment counts for every location in this PR where you use this type.

Reply:
resolved